outlook/src/taskpane/components/Main/Main.tsx

```diff
@@ -161,6 +173,22 @@ class Main extends React.Component<{}, MainState> {
         return {email: email, displayName: displayName}
     }
 
+    private getTranslations = () => {
+        this.setState({translationsLoading: true});
+        sendHttpRequest(HttpVerb.POST, api.baseURL + api.getTranslations, ContentType.Json,
+                this.context.getConnectionToken(), {
```

Comment:
Linting: isn't it some indentation issue ?

Reply:
Yes, corrected!